src/main/java/eu/darkbot/api/entities/utils/Area.java

```diff
@@ -0,0 +1,15 @@
+package eu.darkbot.api.entities.utils;
+
+import java.awt.Shape;
+
+public interface Area extends Shape {//?
```

Comment:
```suggestion
public interface Area {
```
Just area, the interface could have different implementations, some square, some not